daprdocs/content/en/reference/components-reference/supported-pubsub/_index.md

```diff
@@ -9,6 +9,12 @@ aliases:
 no_list: true
 ---
 
+The following table lists publish and subscribe brokers supported, at various levels, by the Dapr pub/sub building block. [Learn how to set up different brokers for Dapr publish and subscribe.]({{< ref setup-pubsub.md >}})
+
 {{< partial "components/description.html" >}}
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
Why add this note here?

Reply:
ope, I added to the wrong doc. I'll move.